web/api/v6/report_server.thrift

```diff
@@ -633,4 +640,20 @@ service codeCheckerDBAccess {
   AnalyzerStatisticsData getAnalysisStatistics(1: i64 runId,
                                                2: i64 runHistoryId)
                                                throws (1: shared.RequestFailed requestError),
+
+  // Add a new filter.
+  // PERMISSION: PRODUCT_ACCESS
+  bool addReportFilter(1: string name,
```

Comment:
For the component creation if I remember correctly admin rights are required, shouldn't we use the same for the filter storage?

Reply:
We can leave it like this now and if the users are asking for it, then we introduce a product specific setting to restrict the filter configuration only to product admins.